src/dotnet/commands/dotnet-help/LocalizableStrings.resx

```diff
@@ -268,7 +268,7 @@
     <value>Path to additional deps.json file.</value>
   </data>
   <data name="InstallDefinition" xml:space="preserve">
-    <value>Installs an item into the development environment.</value>
+    <value>Installs an item in the development environment.</value>
```

Comment:
Looks like `UpdateDefinition` was removed and the `InstallDefinition` was changed.

Reply:
Which would mean we need to check that `update` is showing up in `dotnet --help` too.

Reply:
I changed both https://github.com/dotnet/cli/pull/8794/files#diff-fb9c7630de9fa4ddaf4b225bed103f6dR124

It is called CommandDescription in the beginning (align with the name in uninstall)

Reply:
Right, but these are the help strings for the root help text coming from `dotnet help`.  This one is used here: https://github.com/dotnet/cli/blob/master/src/dotnet/commands/dotnet-help/HelpUsageText.cs#L31.

I don't see a corresponding change to add `update {LocalizableStrings.UpdateDefinition}`.

Reply:
On an aside: these commands really should be in alphabetical order :/

Reply:
I think the problem is I didn't update help text at all. Adding it

Reply:
I am not sure about the alphabetical order tho. The current order is almost a flow of an app.

Reply:
👍 I'd revert this particular change, though. "into" makes sense for install and "in" makes sense for update (the tool is already *in* the environment).

Reply:
Yeah, don't need to solve that with these changes; I just think it's something we should discuss with Kathleen when and if we do the verb/noun reversal.  There's a lot of commands that aren't in an everyday use-flow and having them in order of potential usage is hard for newly added commands to be positioned properly.  Right now the ordering just seems very haphazard.

Most command line tools that have subcommands tend to list them alphabetically.

Reply:
Now it should be good

Reply:
install into is reverted. And help text for update is added

Reply:
Sorry still missing one